plugins/modules/source_control/gitlab/gitlab_protected_branch.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Werner Dijkerman (ikben@werner-dijkerman.nl)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_protected_branch
+short_description: (un)Marking existing branches for protection in Gitlab for an project.
```

Comment:
```suggestion
short_description: (un)Marking existing branches for protection
```
From the module name, I think it's clear that this is about GitLab. If you want a longer description, you should put it in the module's description itself, not here :)